css/basscss.css

```diff
@@ -56,54 +68,17 @@ textarea {
   padding: .5rem .5rem;
 }
 
-.fieldset-reset {
+fieldset {
```

Comment:
I'd keep this a class - especially as this is a library. I think default fieldset stylings can be useful. By setting padding and margin to 0 I feel like this implies you'd never add margins or padding to a fieldset - which I don't find to be true. Would rather add a class to reset it.